Converters/Rhino/Speckle.Converters.RhinoShared/ToSpeckle/Raw/HatchToSpeckleConverter.cs

```diff
@@ -32,12 +36,38 @@ public SOG.Region Convert(RG.Hatch target)
     ICurve boundary = _curveConverter.Convert(rhinoBoundary);
     List<ICurve> innerLoops = rhinoLoops.Select(x => _curveConverter.Convert(x)).ToList();
 
+    // create display mesh from region loops
+    var brep = RG.Brep.TryConvertBrep(target);
+    var displayMesh = GetBrepDisplayMesh(brep);
+    var displayValue = new List<SOG.Mesh>();
+    if (displayMesh != null)
+    {
+      displayValue.Add(_meshConverter.Convert(displayMesh));
+    }
+
     return new SOG.Region
     {
       boundary = boundary,
       innerLoops = innerLoops,
       hasHatchPattern = true,
       units = _settingsStore.Current.SpeckleUnits,
+      displayValue = displayValue
     };
   }
+
+  private RG.Mesh? GetBrepDisplayMesh(RG.Brep brep)
```

Comment:
instead of using this method (looks copied from brep converter, which we aren't using anymore), maybe add a method to handle RG.Breps instead to this class:
https://github.com/specklesystems/speckle-sharp-connectors/blob/dev/Converters/Rhino/Speckle.Converters.RhinoShared/ToSpeckle/Meshing/DisplayMeshExtractor.cs